mcp-server/polaris_mcp/server.py

```diff
@@ -540,6 +541,10 @@ def _resolve_package_version() -> str:
 
 def main() -> None:
     """Script entry point."""
+    # Load configuration from file if specified, otherwise look for .env
```

Comment:
nit: I guess this comment isn't necessary, as the code is quite expressive already.